distribution/src/main/resources/docker-compose/docker-compose.yml

```diff
@@ -260,22 +260,6 @@ services:
       reindexing:
         condition: service_completed_successfully
 
-  live-ingester:
```

Comment:
I would delete it at least for now as we probably want to stop this app and run it locally anyway and it is hard to support both localhost env and docker env when using wiremock

Reply:
If you're proposing a different way of working then can you document the steps to run the application in a dev env? I know for example that this is the way @azakrzewski-hy uses to test the app.

Reply:
I also often run the app in docker, maybe there is other way to adjust d-c - I will think about it 🤔

Reply:
Ok - as discussed let's split this into a separate PR to discuss when you're back. We also thought that we will need to cope with making changes to several apps at once (live ingester, bulk ingester, prediction handler).

Reply:
@Pawel-608 @tpage-alfresco I've prepared solution proposition to this problem - we can discuss it here: https://github.com/Alfresco/hxinsight-connector/pull/76
New approach basically requires to use different d-c to run the env without Ingester service.

Reply:
Thanks @krdabrowski - marking this as resolved.